admin/javascript/src/components/grid-field/row.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import SilverStripeComponent from 'silverstripe-component';
+
+class GridFieldRowComponent extends SilverStripeComponent {
+
+  render() {
+    return (
+      <li className="grid-field-row-component [ list-group-item ]">{this.props.children}</li>
```

Comment:
Why are we using ul/li and not tables? Makes it impossible (or at least very difficult) to have a whole column be the right width later on.